cdap-ui/app/cdap/components/ConfigurationGroup/index.tsx

```diff
@@ -106,36 +114,98 @@ const ConfigurationGroupView: React.FC<IConfigurationGroupProps> = ({
     inputSchema,
   };
 
+  // Used to keep track of error messages that found a widget
+  // required to identify errors that did not find a widget,
+  // they will be marked as orphan errors.
+  const usedErrors = {};
```

Comment:
You should not do this, as React.FC has no guarantee about computed variables like this between render.